packages/opentelemetry-exporter-prometheus/src/PrometheusExporter.ts

```diff
@@ -72,7 +72,11 @@ export class PrometheusExporter implements MetricExporter {
     ).replace(/^([^/])/, '/$1');
 
     if (config.preventServerStart !== true) {
-      this.startServer().then(callback);
+      this.startServer()
+        .then(callback)
+        .catch(err => {
```

Comment:
This catch does nothing. You have to await the promise or return it to have the error bubble up to the caller.

Reply:
returned!